whole-tale/gwvolman
# Refactor publishing tests

## Summary
I separated out the Zenodo and DataONE publishing tests into their own files. I also added a few tests for the DataONE metadata file. I grouped the DataONE tests together in their own folder to match the structure of the src directory.

This should make it easier to segment tests for new things (for example rdf processing in upcoming features) and find weaknesses in existing ones (only caught one bug in the refactor).

### Test Coverage

The overall coverage decreased, I'm guessing mostly due to making the tests a little more granular (on the DataONE side). In particular the EML tests (which weren't being truly tested before).

### To Test

1. Check this PR out
2. Run the unit tetssts
3. Make sure they pass
